CMakeLists.txt

```diff
@@ -69,6 +69,9 @@ elseif(WIN32)
   include(Win32Configuration)
 elseif(UNIX AND (NOT APPLE))
   include(LinuxConfiguration)
+  if(NOT OPENELEC)
+    include(InstallLinuxDesktopFile)
+  endif(NOT OPENELEC)
```

Comment:
Sorry if i was unclear about this :)
But the `InstallLinuxDesktopFile.cmake` should be included from `LinuxConfiguration.cmake` as it's part of linux.
There is not problem making the OPENELEC check in `LinuxConfiguration.cmake` as we already make other checks for Linux related stuff ie X11 there.
Embedded is indeed Linux, but this would not appy for embedded (although adding shorcut wouldn't harm, but would be useless).

Reply:
Ah, okay :)